app/assets/stylesheets/admin.scss

```
@@ -131,7 +133,7 @@ $table-header:   #ecf1f6;
       border: $table-border;
 
       &:hover {
-        background: adjust-color(darken($table-background, 2%), $red: -4, $green: -1);
+        background: adjust-color(color.adjust($table-background, $lightness: -2%), $red: -4, $green: -1);
```

`adjust-color` followed by `color.adjust` sounds a bit strange :thinking:. Maybe we could do something like:

```suggestion
        background: color.adjust($table-background, $lightness: -2%, $red: -4, $green: -1);
```